### PR TITLE
merge from develop

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

**Review by Cheephot — APPROVED** [review]

File: src/ImageService.Business/ImageService.Business.csproj [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  <ItemGroup>
+    <ProjectReference Include="..\ImageService.Data\ImageService.Data.csproj" />
+    <ProjectReference Include="..\ImageService.Mappers\ImageService.Mappers.csproj" />
+    <ProjectReference Include="..\ImageService.Models.Db\ImageService.Models.Db.csproj" />
+    <ProjectReference Include="..\ImageService.Models.Dto\ImageService.Models.Dto.csproj" />
+    <ProjectReference Include="..\ImageService.Validation\ImageService.Validation.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
```
Comment (BelehovEgor) [inline src/ImageService.Business/ImageService.Business.csproj:17]: What is this?

File: src/ImageService.Data/ImageMessageRepository.cs [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+using LT.DigitalOffice.ImageService.Data.Interfaces;
+using LT.DigitalOffice.ImageService.Data.Provider;
+using LT.DigitalOffice.ImageService.Models.Db;
+
+namespace LT.DigitalOffice.ImageService.Data
+{
+  public class ImageMessageRepository : IImageMessageRepository
+  {
+    private readonly IDataProvider _provider;
+    public ImageMessageRepository(IDataProvider provider)
```
Comment (BelehovEgor) [inline src/ImageService.Data/ImageMessageRepository.cs:13]: add line

File: src/ImageService.Models.Dto/ImageService.Models.Dto.csproj [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
```
Comment (BelehovEgor) [inline src/ImageService.Models.Dto/ImageService.Models.Dto.csproj:7]: what is this?

**Review by BelehovEgor — APPROVED** [review]

**Review by VLiamin — APPROVED** [review]

File: src/ImageService.Business/ImageService.Business.csproj (hunk 1 above)
Comment (MariyaSin) [reply src/ImageService.Business/ImageService.Business.csproj:17]: created task

File: src/ImageService.Data/ImageMessageRepository.cs (hunk 2 above)
Comment (MariyaSin) [reply src/ImageService.Data/ImageMessageRepository.cs:13]: created task

File: src/ImageService.Models.Dto/ImageService.Models.Dto.csproj (hunk 3 above)
Comment (MariyaSin) [reply src/ImageService.Models.Dto/ImageService.Models.Dto.csproj:7]: created task

**Pull request merged**